client/components/QuestionsAndAnswers/QAndAComponents/Answer.jsx

```diff
@@ -58,15 +59,15 @@ const Answer = ({ answer }) => {
             <Typography variant="caption">|</Typography>
           </Grid>
           <Grid item className={classes.textSpacing}>
-            <Helpful helpfulness={answer.helpfulness}/>
+            <Helpful helpfulness={answer.helpfulness} questionId='NA' answerId={answer.id}/>
```

Comment:
Wondering why questionId is passed down as props, but then on line 62, it is just set to 'NA'.  Doesn't appear that the questionId props from line 30 is used?